practice-app/app/views.py

```diff
@@ -242,3 +252,48 @@ def worldTime():
                 languageCode=world_time.languageCode,
             )
         return render_template("worldtime.html")
+
+@app.route("/pokemon", methods=["GET", "POST"])
+def PokemonPage():
+    if request.method == "GET":
+        return render_template("pokemon.html")
+    else:
+        pokemon_name = request.form.get("poke_name")
+        url = f"https://pokeapi.co/api/v2/pokemon/{pokemon_name.lower()}"
```

Comment:
What happens when you press find pokemon button without typing anything? Did you test that case? :)

Reply:
Makes sense :) , fixed.